### PR TITLE
Simulator Output Modification

### PR DESCRIPTION
**Pull request by hamza-sadruddin**
Please have a look at the files for Task_D of the project.(Hamza & Saad) Thanks.

### CODE REVIEW

**Comment by hamza-sadruddin** [comment]
Reminder: Review was requested 3 days ago. On no response in a couple of hours, I will merge this pull request. Thanks.

File: README.md [hunk 1]
```diff
@@ -115,7 +122,7 @@ The same steps should be followed for receiver and subnet tests as we are showin
    - To run the simulation, type in the terminal
```
Comment (abhinavachanta) [inline README.md:133]: No information/steps provided on how to check the output of task 4. Pls add.

Comment (abhinavachanta) [reply README.md:133]: Changes to be done in installation guide: 
After modularizing the components as submodule, steps are not updated on how to clone the project along with submodules. Pls add the steps

**Review by abhinavachanta — CHANGES_REQUESTED** [review]
Pls check the inline comments in readme file

File: README.md (hunk 1 above)
Comment (hamza-sadruddin) [reply README.md:133]: Done

File: test/src/receiver/receiver.cpp [hunk 2]
```diff
@@ -76,7 +82,7 @@ using logger_top=cadmium::logger::multilogger<log_messages, global_time>;
 /********************************************/
 /****** APPLICATION GENERATOR *******************/
 /********************************************/
-string input_data_control = "../test/data/receiver_input_test.txt";
+string input_data_control = "test/data/receiver_input_test.txt";
```
Comment (abhinavachanta) [inline test/src/receiver/receiver.cpp:85]: Define a constant using #define at the top and use the #define value. Do not hardcode.
Do this in all test files where ever applicable.

**Review by abhinavachanta — CHANGES_REQUESTED** [review]
Check comments I made in receiver test file

File: test/src/receiver/receiver.cpp (hunk 2 above)
Comment (hamza-sadruddin) [reply test/src/receiver/receiver.cpp:85]: I guess we were not supposed to touch the simulator. The instructor told us to leave the files as such. Also, none of us took this file for any of the tasks.

Comment (abhinavachanta) [reply test/src/receiver/receiver.cpp:85]: Ok

**Review by abhinavachanta — APPROVED** [review]
Approved.

**Review by jrajeev459 — APPROVED** [review]
sorry guys for the delay. i was not able to give the feedback on time because i was busy with my part of the project and other exams. approved. thank you.

**Pull request merged**